src/components/instance-configurator/InstanceConfigurator.jsx

```diff
@@ -15,6 +17,8 @@ const styles = () => ({
 class InstanceConfigurator extends React.Component {
     state = {
         tableKey: Math.random(),
+        dialogOpen: false,
```

Comment:
Proposal: Remove `dialogOpen` and use `toDelete` to determine if the dialog should be open or not.

Reply:
like it